### PR TITLE
Deploy updates

### PR DESCRIPTION
**Pull request by portante**
Updates the deployment on the `b2.1` branch to include auto-tagging and copying of the SHA1 remotely so that we can verify what was deployed.

See also https://docs.npmjs.com/cli/v7/commands/npm-version#description and the description about how to use the `preversion`, `version`, and `postversion` scripts.

### CODE REVIEW

**Review by dbutenhof — COMMENTED** [review]
I'm not familiar with the npm `version` stuff, and I don't know where that came from since it's not on `main` ... but it looks plausible enough. The Ansible changes look straightforward. Mostly, it seems that all of this should probably be on `main`, too, since none of this looks specific to the old ES1 dashboard.

File: package.json [hunk 1]
```diff
@@ -1,13 +1,19 @@
 {
   "name": "pbench_dashboard",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (aquibbaig) [inline package.json:3]: I am not sure but the version in my deployed dashboard shows `2.1.1-0`

**Review by aquibbaig — APPROVED** [review]
Looks good!

**Review by gurbirkalsi — APPROVED** [review]

**Review by anishaswain — APPROVED** [review]
LGTM!

**Review by dbutenhof — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (portante) [reply package.json:3]: How did you deploy it?

**Comment by portante** [comment]
I'll file another PR to forward port these changes to the `main` branch.

File: package.json (hunk 1 above)
Comment (aquibbaig) [reply package.json:3]: @portante I deployed it on localhost, changing some parts of main.yml

**Pull request merged**